scripts/start.js

```diff
@@ -364,7 +369,8 @@ function getUserInfo () {
         resolve({
           userName: username.slice(0, username.indexOf('@')),
           ssoToken: body.access_token,
-          domain: domain
+          domain: domain,
+          userId: userId,
```

Comment:
@mareklibra if userId is missing then `options` will be hidden, I also added this description to start.js.